tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -1323,7 +1327,7 @@ func allowRuleSync() {
 
 func allowPrometheusSync() {
 	log.Info("Sleeping to allow prometheus to record metrics...")
-	time.Sleep(15 * time.Second)
+	time.Sleep(30 * time.Second)
```

Comment:
we had just turned this down to 15 in an effort to speed up tests (prom scrape time was reduced to 1s). Have you seen issues around timing here?

Reply:
It failed twice, once on circle and once on prow. 30s seems to pass.